core/vm/logger.go

```diff
@@ -177,7 +180,10 @@ func (l *StructLogger) CaptureState(pc uint64, op OpCode, gas, cost uint64, scop
 		}
 	}
 	// Copy a snapshot of the current storage to a new container
-	var storage Storage
+	var (
+		storage Storage
+		proof   [][]byte
```

Comment:
prefer renaming to `storageProof`

Reply:
Don't we need tx list and block property information?

Reply:
This is a temporary variable in a function. Are you sure?

Reply:
This is a temporary variable in a function. Are you sure?

Reply:
I think so. because proof can also refer to `account proof`. using `storageProof` can avoid ambiguity

Reply:
Changed.